src/scss/main.scss

```diff
@@ -499,6 +594,18 @@ $curve-springy: cubic-bezier(0.43, 0.48, 0.62, 1.07);
   display: block;
 }
 
+[data-braintree-id="methods-container"] {
```

Comment:
[Related converation](https://github.com/braintree/braintree-web-drop-in/pull/70#discussion_r101663983)

There was only one other property defined under here after cleaning up `braintree-methods-container--inactive`. This property was also defined in `.braintree-loader__indicator`.

So we removed this all together.